homeassistant/components/tradfri/utils.py

```diff
@@ -0,0 +1,40 @@
+"""Util functions for IKEA Tradfri."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from functools import wraps
+import logging
+from typing import Any
+
+from pytradfri.command import Command
+from pytradfri.error import PytradfriError
+
+from .const import ATTR_MAX_FAN_STEPS
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def _from_fan_percentage(percentage: int) -> int:
```

Comment:
If the function is only used in one module it's easier to read the code if the function is defined in that module.

Reply:
Noted, I reverted the changes.